funnel/views/space.py

```diff
@@ -26,7 +26,7 @@ def space_data(space):
         'timezone': space.timezone,
         'start': space.date.isoformat() if space.date else None,
         'end': space.date_upto.isoformat() if space.date_upto else None,
-        'status': space.status,
+        'status': space.status.value,
```

Comment:
`'status': space.state.label.name`

Reply:
For API compatibility, maybe we should include both:
```python
'status': space.status.value,
'state': space.state.label.name,
```
And remove `status` once the Android app is updated. (Also applies to the other example above.)